README.md

```diff
@@ -55,18 +59,18 @@ The Obstacle Tower environment runs on Mac OS X, Windows, or Linux.
 Python dependencies (also in [setup.py](https://github.com/Unity-Technologies/obstacle-tower-env/blob/master/setup.py)):
 
 * Python 3.6
-* Unity ML-Agents v0.10
+* Unity ML-Agents 1.x
 * OpenAI Gym
 
 ### Download the environment
 
 | *Platform*     | *Download Link*                                                                     |
 | --- | --- |
-| Linux (x86_64) | https://storage.googleapis.com/obstacle-tower-build/v3.1/obstacletower_v3.1_linux.zip   |
-| Mac OS X       | https://storage.googleapis.com/obstacle-tower-build/v3.1/obstacletower_v3.1_osx.zip     |
-| Windows        | https://storage.googleapis.com/obstacle-tower-build/v3.1/obstacletower_v3.1_windows.zip |
+| Linux (x86_64) | https://storage.googleapis.com/obstacle-tower-build/v4.0/obstacletower_v4.0_linux.zip   |
+| Mac OS X       | https://storage.googleapis.com/obstacle-tower-build/v4.0/obstacletower_v4.0_osx.zip     |
```

Comment:
I get an error trying to download this

Reply:
In the PR I say that this is WIP and binaries are not live yet. Mainly I wanted a review of the changes to the gym wrapper, which are extensive.